.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Learn about CODEOWNERS file format:
 #  https://help.github.com/en/articles/about-code-owners
 
-* @SumoLogic/k8s-collection-team
+* @SumoLogic/k8s-collection-team @astencel-sumo @frankreno @kkujawa-sumo @perk-sumo @pmalek-sumo @sumo-drosiek
```

Comment:
Wait. This will still randomly choose 3 people from a combined set of people (explicitly listed here) and the team, right?

Don't we want to remove the collection team from here to prevent requesting review from people not actively maintaining this repo?

Reply:
Using three random people is a setting for the group, I don't see any such point in codeowners documentation

Reply:
I don't have strong opinion regarding @SumoLogic/k8s-collection-team
@perk-sumo ?

Reply:
I don't have an opinion either but adding code owners in order to automatically do something (add reviewers) that we'll need to manually change anyway doesn't make much sense to me.

Reply:
what do you want to change manually?

Reply:
My understanding is that 3 out of N people from a team will be picked for a review, +all explicitly mentioned?

Reply:
AFAIK

Reply:
but, It can be also dead code, as first the explicit members will be picked up, and because they alfready contains 3 of team members, no additional team member will be added